6-AdvancedScenarios/1-call-api-obo/API/authConfig.js

```diff
@@ -0,0 +1,36 @@
+const authConfig = {
+    credentials: {
+        tenantID: 'Enter_the_Tenant_Id_Here',
+        clientID: 'Enter_the_Application_Id_Here',
+        clientSecret: 'Enter_the_Client_Secret_Here',
+    },
+    metadata: {
+        authority: 'login.microsoftonline.com',
+        discovery: '.well-known/openid-configuration',
+        version: 'v2.0',
+    },
+    settings: {
+        validateIssuer: true,
+        passReqToCallback: true,
+        loggingLevel: 'info',
+        loggingNoPII: true,
+    },
+
+    resources: {
+        downstreamAPI: {
+            endpoint: 'https://graph.microsoft.com/v1.0',
+            scopes: ['User.Read', 'offline_access'],
+        },
+        middleTierAPI: {
+            endpoint: '/api/profile',
+            delegatedPermissions: {
+                scopes: ['access_graph_on_behalf_of_user'],
+            },
+            applicationPermissions: {
+                scopes: ['access_graph_on_behalf_of_user'],
+            },
```

Comment:
```suggestion
```

Reply:
application permissions are not usable with OBO. we should remove everything related to app only tokens